js/widgets/forms/slider.backcompat.js

```diff
@@ -0,0 +1,84 @@
+/*!
+ * jQuery Mobile Slider Backcompat @VERSION
+ * http://jquerymobile.com
+ *
+ * Copyright jQuery Foundation and other contributors
+ * Released under the MIT license.
+ * http://jquery.org/license
+ */
+
+//>>label: Slider
+//>>group: Forms
+//>>description: Deprecated Slider features
+
+( function( factory ) {
+	if ( typeof define === "function" && define.amd ) {
+
+		// AMD. Register as an anonymous module.
+		define( [
+			"jquery",
+			"../widget.backcompat",
+			"./slider" ], factory );
+	} else {
+
+		// Browser globals
+		factory( jQuery );
+	}
+} )( function( $ ) {
+
+if ( $.mobileBackcompat !== false ) {
+
+	$.widget( "mobile.slider", $.mobile.slider, {
+		options: {
+			corners: true,
+			mini: false,
+			highlight: false
+		},
+		classProp: "ui-slider",
+		_create: function() {
+			this._super();
+
+			if ( this.options.mini ) {
+				this._addClass( this.slider, "ui-mini", null );
+			}
+
+			if ( this.options.highlight && this.slider.find( ".ui-slider-bg" ).length === 0 ) {
+				this.valuebg = ( function( slider ) {
+					var bg = document.createElement( "div" );
+					bg.className = "ui-slider-bg " + $.mobile.activeBtnClass;
+					return $( bg ).prependTo( slider );
+				} )( this.slider );
+			}
+
+			if ( this.options.highlight ) {
+				this._setHighlight( this.options.highlight );
+			}
+
+			if ( this.options.corners !== undefined ) {
+				this._setCorners( this.options.corners );
+			}
+		},
+
+		_setHighlight: function( value ) {
+			if ( value ) {
+				this.options.highlight = !!value;
+				this.refresh();
+			} else if ( this.valuebg ) {
+				this.valuebg.remove();
+				this.valuebg = false;
+			}
+		},
+
+		_setCorners: function( value ) {
+			this._toggleClass( this.slider, null, "ui-corner-all", value );
+			this._toggleClass( this.element, null, "ui-corner-all", value );
+		}
+	} );
+
+	$.widget( "mobile.slider", $.mobile.slider, $.mobile.widget.backcompat );
```

Comment:
@arschmitz not sure what you mean here. This code is required to get the backcompat to function properly because the way the slider is styled it doesn't automatically get the required classes for these properties.